src/Model/Translatable/TranslatableMethods.php

```diff
@@ -43,11 +43,11 @@ public function getNewTranslations()
     /**
      * Adds new translation.
      *
-     * @param Translation $translation The translation
+     * @param TranslationsInterface $translation The translation
```

Comment:
Should be `TranslationInterface` (singular)

Reply:
:+1: